catalog/video/video-mpegts-stream/pom.xml

```diff
@@ -219,22 +219,22 @@
                                         <limit>
                                             <counter>INSTRUCTION</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.70</minimum>
+                                            <minimum>0.67</minimum>
```

Comment:
unfortunately, no. The networking code can't be tested in a unit test. At best, it could be tested in an itest, but that is problematic for multicast because it depends on the network configuration/conditions outside of the test server.